app/models/order_detail.rb

```diff
@@ -679,7 +679,13 @@ def assign_price_policy
   end
 
   def assign_actual_price
-    pp = product.cheapest_price_policy(self, time_for_policy_lookup)
+    # If time_for_policy_lookup is nil, then the default date is not set
```

Comment:
Why is this change needed?  if I remove it, `spec/system/admin/manage_order_detail_spec.rb` still passes

Reply:
That's interesting. I did a ton of debugging and updating that was the only way of having `product.cheapest_price_policy` return any policies at all